explore/src/main/scala/explore/tabs/TwoPanels.scala

```diff
@@ -47,16 +46,13 @@ trait TwoPanels {
     ctx:       AppContext[IO]
   ): VdomNode =
     Button(
-      as = <.a,
-      size = Mini,
-      compact = true,
-      basic = true,
-      clazz = ExploreStyles.TileBackButton |+| ExploreStyles.BlendedButton,
-      onClickE = linkOverride[ButtonProps](
-        ctx.pushPage(appTab, programId, Focused.None) >>
-          pv.set(SelectedPanel.Tree)
-      )
-    )(^.href := ctx.pageUrl(appTab, programId, Focused.None), Icons.ChevronLeft)
+      icon = Icons.ChevronLeft,
+      severity = Button.Severity.Secondary,
+      text = true,
+      clazz = ExploreStyles.TileBackButton,
+      onClick = ctx.pushPage(appTab, programId, Focused.None) >>
+        pv.set(SelectedPanel.Tree)
+    ).mini.compact
```

Comment:
This no longer displays the link on hover, but since it usually will only be displayed on smaller devices, they wouldn't usually see the hover link, anyway. It could be switched to a `span` or a tooltip added if we need it.